infra/src/main/resources/db/migration/prod/V61__add_switch_to_ownership_boundary_description_suffix.sql

```diff
@@ -0,0 +1 @@
+alter type layout.location_track_description_suffix add value 'SWITCH_TO_OWNERSHIP_BOUNDARY';
```

Comment:
Mietin tämän tekemistä repeatable-migraationa jolloin se ei konfliktaisi splittausbranchin kanssa, mutta päädyin lopulta sitä vastaan. Syynä se, että jos tämä halutaankin syystä tai toisesta poistaa koskaan, niin sen poistaminen lienee helpompaa näin.

Reply:
Joo ei noita enum-tyyppejä minusta kannata laittaa repeatableiksi. Siitä tulee vain tuskaa jos niitä haluaa joskus käyttää jossain versioidussa